src/com/google/foam/support/views/capstone/messageModel.js

```diff
@@ -0,0 +1,129 @@
+foam.CLASS({
+    package: 'com.google.foam.support.views.capstone',
```

Comment:
place this under the same directory as everyone else. foam/support/view & rename the package name. to fit the path.